test/jdk/javax/accessibility/JSlider/JSliderAccessibilityTest.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright (c) 2019, 2022 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is it a new test or it is based on some old code?

Reply:
fixed